projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/style.scss

```diff
@@ -57,4 +47,51 @@
         line-height: var(--font-title-small);
         color: var(--jp-gray-70);
     }
+
+    &__data {
+        display: flex;
+        align-items: center;
+        font-size: var(--font-body-extra-small);
+        line-height: var(--font-title-small);
+        color: var(--jp-gray-50);
+        font-weight: 500;
+    }
+
+    &__status-icon {
+        display: block;
+        margin-right: 6px;
+    }
+
+    &__status-text {
+        margin-right: 1px;
+        letter-spacing: -0.24px;
```

Comment:
Do we need such specific spacing?

Reply:
It was exactly per the design. I'll leave a comment. 👍